### ISSUE TITLE
-s option not working properly

### ISSUE THREAD

**Issue by pdelisle6180** [chapagain/php-swiss-ephemeris#2]
I am not having any success using the -s option with minutes or other time modifiers.
Example: -n10 -s60m
This should generate 10 outputs separated by 60 minutes in time. The "m" modifier has no effect. Instead I get 10 outputs separated by 60 days instead of 60 minutes. The program ignores the m or any of the other letters after the number. The other options like s, mo and y (for seconds, months, years) don't work either.

**Comment by chapagain** [comment]
Yes, you are right. I tried with -s60m, -s6m, -s6y and all of them didn't work. All of them gave days wise result instead. I have reported this problem to [Astrodienst](https://www.astro.com). Hope they will respond soon and thank you for reporting this issue.

**Comment by pdelisle6180** [comment]
Hi Mukesh,

Thank you for the quick reply.
Interestingly this options works fine when I try it on the Swiss Ephemeris
test page!
How can that be?
http://www.astro.com/swisseph/swetest.htm

Paul

**Comment by pdelisle6180** [comment]
*This could be a version issue. Are you using the latest version of
swetest?*

*I found this on the forum:*
Thoma, the issue had already been resolved. Charles had used an older
version of swetest, these options work only since version 2.05.01

-sN timestep N days, default 1. This option is only meaningful
when combined with option -n.

If an 'y' is appended, the time step is in years instead of days,
for example -s10y for a time step of 10 years.
If an 'mo' is appended, the time step is in months instead of days,
for example -s3mo for a time step of 3 months.
If an 'm' is appended, the time step is in minutes instead of days,
for example -s15m for a time step of 15 minutes.
If an 's' is appended, the time step is in seconds instead of days,
for example -s1s for a time step of 1 second.

**Comment by chapagain** [comment]
I downloaded the "swetest" exectuable file from http://www.astro.com/ftp/swisseph/src/ and updated it with the one present in this repo. But, still the same issue. Can you try at your end as well?

**Comment by pdelisle6180** [comment]
I did the same and it's working fine with the swetest.exe I downloaded
directly from the astro.com site.

**Comment by pdelisle6180** [comment]
Hi Mukesh,

​Same here. I tried the one on their site (swetest) and it's not working
either.
So this means that the problem is on their side.
However is does work when I use (locally) the swetest.exe file compiled for
Windows.

Did you get any reply from you communication with them yet?​

**Comment by chapagain** [comment]
Hi Paul,

I have not received any reply from astro.com of my email to them regarding this issue. 

Today, I have also submitted a question in their forum: https://forum.astro.com/cgi/forum.cgi?num=1527060442

Hope to get some replies there.

**Comment by pdelisle6180** [comment]
Mukesh,

I just got a response from them.
They claim that we are using a older version because these options were not
in the older version.
I will let you know when I learn more.

**Comment by pdelisle6180** [comment]
It didn't work for me either.
I found one that works here:
https://github.com/cyjoelchen/php-sweph
